src/specVersions/v0.0/RicardianContractProcessorImpl.ts

```diff
@@ -62,19 +59,40 @@ export class RicardianContractProcessorImpl implements RicardianContractProcesso
     this.wrappedHelpers.push(name)
   }
 
+  protected registerHelper(name: string, fn: Handlebars.HelperDelegate) {
+    Handlebars.registerHelper(name, fn)
+  }
+
   public getSpecVersion() {
     return {
```

Comment:
for consistency, maybe we should update v0.0 to also have a const at the top of the file, similar to the [v0.1 implementation](https://github.com/EOSIO/ricardian-template-toolkit/blob/40d686c7750d9316f90c52b848c8bcbe48992d68/src/specVersions/v0.1/RicardianContractProcessorImpl.ts#L5)?
```
const implVersion = {
  major: 0,
  minor: 0,
}
```

Reply:
And then just return `implVersion` here